lib/middleware/authorizeReview.js

```diff
@@ -0,0 +1,17 @@
+const Review = require('../models/Review');
+
+module.exports = async (req, res, next) => {
+  const review = await Review.getById(req.params.id);
+  console.log(review);
+  console.log(req.params);
+  console.log(req.user);
+  try {
+    if (req.user.id !== review.userId) {
```

Comment:
i think technically, this was also supposed to check for admin -- I'm not going to dock you on it but its easy enough to add --
```js
if (req.user.email !== 'admin' || req.user.id !== review.userId) {...}
```